### ISSUE TITLE
Upload an attachment without access right

### ISSUE THREAD

**Issue by anandkansagra** [odoo/odoo#29048]
Impacted versions:
10, 11, 12

Steps to reproduce:
Give read access to `ir.attachment` object and upload a document

Current behaviour:
There seems an error if no access right for `create` in the `ir.attachment` object.

Expected behavior:
If there are no access right for `create` in the `ir.attachment` object then `Attachment` button should be invisible
Video/Screenshot link (optional):

1) Access right for `ir.attachment` object
![1](https://user-images.githubusercontent.com/45279628/49058821-b50f6280-f22b-11e8-96b9-176176e346ad.png)

2) There seems no `Create` right for the `ir.attachment` object then also user can add the `Attachment`
![2](https://user-images.githubusercontent.com/45279628/49058850-e0924d00-f22b-11e8-999b-912637b05122.png)

3) There seems an error if user will add the attachment without an access right
![3](https://user-images.githubusercontent.com/45279628/49058885-f99afe00-f22b-11e8-836a-7cceabe0c85a.gif)

**Comment by pedrobaeza** [comment]
I agree that this is not very UX friendly, but the permission check avoids to do something incorrect (in security terms). You are also deforming a bit the standard system giving only read access that way. I doubt this can't be done without needing extra RPC calls, so this might be discarded, but I let framework team the last word.

**Comment by C3POdoo** [comment]
Dear @anandkansagra,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
